HIP/hip-991.md

```diff
@@ -87,9 +87,7 @@ We propose adding a fixed fee mechanism to the Hedera Consensus Service (HCS) fo
 * Fees can be changed (updated or removed) in a topic with an update topic transaction signed by the Fee Schedule Key.
 * Topics can have a Fee Schedule Key set at creation time.
 * The Fee Schedule Key can manage fee updates for the topic.
-* The Fee Schedule Key follows the same "Lower privilege key" rules introduced by [HIP-540](https://hips.hedera.com/hip/hip-540), i.e:
-  * Only the Admin Key should be able to remove the Fee Schedule Key.
-  * The Fee Schedule Key can change itself to another valid or unusable key.
+* The Fee Schedule key can be updated according to the same rules that currently apply to the Submit key.
```

Comment:
We should add that if the Fee Schedule Key is updated, the new one must sign the transaction. This will become mandatory once we implement HIP-540 for the Hedera Consensus Service. It will become a breaking change if we do not enforce this from the beginning.

Reply:
Done